BUILDING.md

```diff
@@ -1,21 +1,17 @@
-# Build-System CLI
+# Getting started / building from source
 
-## Non-interactive
-```
-python build.py -h, --help
+1. clone the source code from the [J2V8 GitHub repository](https://github.com/eclipsesource/J2V8)
```

Comment:
Previously I added https://github.com/eclipsesource/J2V8/pull/327#pullrequestreview-55968060 and @irbull found downloading a node version not a good separation of concerns. But I don't see what is the problem, actually.

Now we have all build options that include node already in a hard-coded version. Node yes/no could be an additional question at startup, followed by typing a version after 'yes'. This can of course be added in a follow-up PR, doesn't need to be in this one.